### ISSUE TITLE
New process tabs randomly fail to reach app directory

### ISSUE THREAD

**Issue by plehoux** [EtienneLem/divide#4]
Sometimes, `cd` randomly  fails. Any idea why?

**Comment by EtienneLem** [comment]
I’ve never experience a `cd` issue, but could it be because your Terminal app isn’t focused anymore?

**Comment by plehoux** [comment]
I'm wondering if a small timeout in between your apple script call to open a tab and the `cd` call could mitigate this. I will try it out.

**Comment by EtienneLem** [comment]
Yeah, this is pretty old school stuff. I was thinking that (one day) I could rewrite it using Apple’s AppleScript JavaScript port and give it some more feature:
- `.divide` for moar local options
- Setting tab name to current process
